Weekly hb od tweak

Calumm07 opened the pull request:
Changing geog names, splitting cumulative file and adding pillar columns

Calumm07 committed:
adding pillars to weekly file

iainma09 committed:
Changes to cumulative sections to add unknown age and/or sex to outputs. Simplified names of intermediate tables as becomming too long. Re-ordered tables to help make checking of age/sex splits more easy to follow. Added QF for unknowns.

iainma09 committed:
add d or :  to CrudeRateQF, consistent with weekly SIMD

Calumm07 committed:
adding cases to variable names

Calumm07 committed:
splitting cumulative hb and la

iainma09 committed:
temporary changes, still having issues with weekly unknowns

iainma09 committed:
re-order age/sex template dataframe so close to cumulative structure, join the intermediate dataframes to this and then calculate cumulatives once combined

Jen-e2 committed:
Update wastewater source and metadata

Jen-e2 committed:
update Ethnicity sentence

Calumm07 committed:
Merge pull request #31 from Public-Health-Scotland/wastewater_updates

Wastewater updates

iainma09 committed:
Carehomes, remove timestapm in csv output
SIMD simplified the steps and renamed variables for consistency. Corrected chronological order so earliest to latest
AgeSexCases, added unknowns for age or sex and age AND sex. Renamed variables for consistency

iainma09 committed:
re-position the od_dates to the start

iainma09 reviewed (commented):
All looks good, a few wee things though.  
In the cumulative outputs, I think you should change  "TotalCases" to "CumulativePositiveCases" to be consistent with Weekly equivalents (and the Age/Sex and deprivation demographic outputs.)
The ordering in the weekly outputs should be TotalTests, PositiveTests, Cases, (Admin) etc. 
I don't think we were to include the pillar testing data the CouncilArea outputs- weekly or cumulative? but I could be wrong
The PositiveTests and HospitalAdmin in the weekly outputs have blank cells, but its zeros in the cases section. Should we just replace blanks with zeroes (or vice versa) at this stage until data suppression is sorted out

Calumm07 committed:
changed cumulative variable names, the ordering of weekly outputs, removed pillars from LA files

iainma09 committed:
Merge pull request #26 from Public-Health-Scotland/od_agesex_weekly_nulls_im

Od agesex weekly nulls im

Calumm07 committed:
replacing blanks with 0s

iainma09 committed:
update geography and simd  postcode lookups to latest published version

iainma09 committed:
Hashtag out admissions section from weekly geography (hold it until confirmed that it should be removed). Add weekly national admissions to admissions transfer script

iainma09 committed:
remove Total and Postive LFD tests from weekly geography
reorder weekly admissions, so Geography at the start

iainma09 committed:
simplify age sex unknowns to one aggregated value for cummulative and one per week for weekly
standardise simd outputs so clearer that data relate to cases

iainma09 committed:
arrgh

iainma09 committed:
tweaks to cases by simd and age/sex

iainma09 committed:
update carehomes, admssions, ethnicity so in correct format
remove LFDTests from CA outputs

iainma09 committed:
adding geography code to admissions, renaming and creating an od_version
adding geography code to care home admissions,
arrange SIMD in orrect chronological order

iainma09 committed:
updates to names, adding occupancy to weekly admissions

iainma09 committed:
Ethnicity rename location code
Geography= rename geography variables
agesex = rename cases
simd-= renames caqses

iainma09 committed:
streamline loading of cases so consistent with gepgraphy names.

iainma09 committed:
source data transfer scripts  for open data

iainma09 committed:
start to stream lineopen data data transfer process.
Create new folder for Open Data files
admissions update to point scotland populations to i_population v2
care homes update carehomes file path output
cases update to point scotland populations to i_population v2
geography rename cummulative geography  variables & update output csv name
agesex start to point population demographics to new populations look_up file

iainma09 committed:
Admissions, start to reorganise so that occupancy joins admissions. this will create blank cells pre Sep 2020 so need a QF for occupancy. Still to be tested as awaiting files
Geography rename and reorganise variables
Age Sex rename and reorganise variables
Simd rename and reorganise variables

iainma09 committed:
creating a method to save out open data files to an archive. still to be completeted

iainma09 committed:
final tweaks and edits

iainma09 committed:
ensure all outputs have blank cells as blank
typo in age sex cumulative
tidy-up unneeded hash-tagged code in all outputs

iainma09 committed:
re organise so that OP pathways are at the start so that they are loaded for existing transfers where we have added to rather than create from scratch

iainma09 committed:
Merge branch 'main' into weekly_hb_od_tweak

iainma09 committed:
Admissions: There is an issue with the date filter for occupancy (filtering before changing the date formats). Move line 73 to 76 (after the mutate has been completed) and this sorts that out.
Weekly_simd_cases_od:
Change line 184 from CumulativePositiveCases to CumulativePositive
Change line 195 from PositiveLastSevenDays to WeeklyPositiveCases

iainma09 committed:
create a data an archiving step so that it moves all exsiting files from opendata folder into a new archive folder. THis is to be done before the  data transfer creates new files
age/sex cases: simplify the creating of age/sex and unknowns equivalent which is then used to create the different breakdowns.

Calumm07 committed:
Merge branch 'new_structure_cm' into weekly_hb_od_tweak
